### PR TITLE
Sync with v.0.2.0

### PR DESCRIPTION
**Pull request by cesco69**
Added new options (minwidth, minheight and splitscreen) into v.0.2.0 see https://github.com/nextapps-de/winbox#options

### CODE REVIEW

File: types/winbox/index.d.ts [hunk 1]
```diff
@@ -63,10 +63,13 @@ declare namespace WinBox {
         index?: number;
         left?: string | number;
         max?: boolean;
+        minheight?: number;
+        minwidth?: number;
```
Comment (peterblazejewicz) [inline types/winbox/index.d.ts:67]:
```suggestion
        minheight?: string | number;
        minwidth?: string | number;
```

> supports units "px" and "%")

**Review by peterblazejewicz — COMMENTED** [review]
I've missed this, sry:
```ts
// Type definitions for winbox 0.1
```
should read now:
```ts
// Type definitions for winbox 0.2
```
you can apply changes yourself within PR using suggest feature, thx!
https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@cesco69 thx!

**Comment by cesco69** [comment]
Ready to merge

**Pull request merged**